lib/src/isolate_manager/isolate_manager_io.dart

```diff
@@ -0,0 +1,128 @@
+import 'dart:async';
+import 'dart:developer';
+import 'dart:isolate';
+
+import 'package:dash_kit_network/dash_kit_network.dart';
+import 'package:dash_kit_network/src/isolate_manager/isolate_manager.dart';
+
+/// The `IsolateManager` class manages a separate isolate for executing tasks
+/// asynchronously. It implements the `IsolateManagerInterface` interface,
+/// which is not shown here.
+// ignore: prefer-match-file-name
+class IsolateManager extends IsolateManagerInterface {
```

Comment:
We should use the same name for io and web due to conditional import?

Reply:
Fixed it.